doc/MANUAL.adoc

```diff
@@ -792,9 +792,9 @@ WARNING: Do not enable this option unless you are aware of these caveats:
     If true (which is the default), ccache will include the current working
     directory (CWD) in the hash that is used to distinguish two compilations
     when generating debug info (compiler option `-g` with variations).
-    Exception: The CWD will not be included in the hash if
-    <<config_base_dir,*base_dir*>> is set (and matches the CWD) and the
```

Comment:
I removed the `base_dir` reference, since it looks like it's not needed at all for this to happen.

Reply:
Look OK.